### ISSUE TITLE
[BUG] v3.3.4 reload_routes --> WARNING:core:utimer_ticker: utimer task

### ISSUE THREAD

**Issue by vtzan** [OpenSIPS/opensips#3022]
**OpenSIPS version you are running**
```
version: opensips 3.3.4 (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, Q_MALLOC, F_MALLOC, HP_MALLOC, DBG_MALLOC, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll, sigio_rt, select.
git revision: 4569c3c3f
main.c compiled on 16:19:05 Feb 28 2023 with gcc 12
```

**Describe the bug**
If I try to use reload_routes
`curl --location --request POST 'http://10.100.0.130:8080/json' --header 'Content-Type: application/json' --data-raw '{"jsonrpc": "2.0","method":"reload_routes","id": 11}'    `    

then

```
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 50 ms ago (now 20230 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 50 ms ago (now 20230 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 150 ms ago (now 20330 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 150 ms ago (now 20330 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 250 ms ago (now 20430 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 250 ms ago (now 20430 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 350 ms ago (now 20530 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 350 ms ago (now 20530 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 450 ms ago (now 20630 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 450 ms ago (now 20630 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 500 ms ago (now 20730 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 550 ms ago (now 20730 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 550 ms ago (now 20730 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 600 ms ago (now 20830 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 650 ms ago (now 20830 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 650 ms ago (now 20830 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 690 ms ago (now 20920 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 740 ms ago (now 20920 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 740 ms ago (now 20920 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 780 ms ago (now 21010 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 830 ms ago (now 21010 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 830 ms ago (now 21010 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 880 ms ago (now 21110 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 930 ms ago (now 21110 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 930 ms ago (now 21110 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 980 ms ago (now 21210 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1030 ms ago (now 21210 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1030 ms ago (now 21210 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1080 ms ago (now 21310 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1130 ms ago (now 21310 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1130 ms ago (now 21310 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1170 ms ago (now 21400 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1220 ms ago (now 21400 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1220 ms ago (now 21400 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1270 ms ago (now 21500 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1320 ms ago (now 21500 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1320 ms ago (now 21500 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1360 ms ago (now 21590 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1410 ms ago (now 21590 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1410 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <rtpengine-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <dlg-reinvite-pinger> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <dlg-options-pinger> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <dlg-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <nh-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <nh-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <nh-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <nh-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <pg-chk-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <EBR timeout> already scheduled 960 ms ago (now 21590 ms), skipping execution
 WARNING:core:timer_ticker: timer task <ul-refresh-timer> already scheduled 960 ms ago (now 21590 ms), skipping execution
 WARNING:core:timer_ticker: timer task <clstr-heartbeats-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <tm-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <tm-timer> already scheduled 960 ms ago (now 21590 ms), delaying execution
 WARNING:core:timer_ticker: timer task <blcore-expire> already scheduled 960 ms ago (now 21590 ms), skipping execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1450 ms ago (now 21680 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1500 ms ago (now 21680 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1500 ms ago (now 21680 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1530 ms ago (now 21760 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1580 ms ago (now 21760 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1580 ms ago (now 21760 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1620 ms ago (now 21850 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1670 ms ago (now 21850 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1670 ms ago (now 21850 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1700 ms ago (now 21930 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1750 ms ago (now 21930 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1750 ms ago (now 21930 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1790 ms ago (now 22020 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1840 ms ago (now 22020 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1840 ms ago (now 22020 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1890 ms ago (now 22120 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1940 ms ago (now 22120 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 1940 ms ago (now 22120 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <cl-sync-check> already scheduled 1990 ms ago (now 22220 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 2040 ms ago (now 22220 ms), delaying execution
 WARNING:core:utimer_ticker: utimer task <tm-utimer> already scheduled 2040 ms ago (now 22220 ms), delaying execution
 WARNING:core:handle_timer_job: utimer job <tm-utimer> has a 2090000 us delay in execution
 WARNING:core:handle_timer_job: utimer job <tm-utimer> has a 2120000 us delay in execution
 WARNING:core:handle_timer_job: utimer job <cl-sync-check> has a 2070000 us delay in execution
 WARNING:core:handle_timer_job: timer job <rtpengine-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <dlg-reinvite-pinger> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <dlg-options-pinger> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <dlg-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <nh-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <nh-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <nh-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <nh-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <pg-chk-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <EBR timeout> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <ul-refresh-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <clstr-heartbeats-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <tm-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <tm-timer> has a 1670000 us delay in execution
 WARNING:core:handle_timer_job: timer job <blcore-expire> has a 1670000 us delay in execution
```

**Comment by bogdan-iancu** [comment]
@vtzan , this should be addressed by the fce0eaeb241866d0c42db41160ed454be59fd7ed and 35bb59e0949efd4166d68643de6c923313a86c0c commits, let's see how they will do for 3.4

**Comment by bogdan-iancu** [comment]
fce0eaeb241866d0c42db41160ed454be59fd7ed is backportable , once we get more testing on it, to be sure it has no issues. The second one is too  intrusive for a backport.

**Comment by volga629-1** [comment]
I been testing fix and everything works as expected. Thank you.

**Comment by bogdan-iancu** [comment]
backports are now available all the way to 3.1, closing here

**Comment by vtzan** [comment]
> backports are now available all the way to 3.1, closing here

Great! Thanks alot!!!
